src/components/Components/ComponentDetails/tabs/ComponentLatestBuild.tsx

```diff
@@ -12,25 +12,22 @@ import {
   Spinner,
   Button,
 } from '@patternfly/react-core';
+import { useImageProxyHost } from '~/hooks/useImageProxyHost';
+import { useImageRepository } from '~/hooks/useImageRepository';
 import { useLatestSuccessfulBuildPipelineRunForComponentV2 } from '~/hooks/useLatestPushBuildPipeline';
 import { useTaskRunsForPipelineRuns } from '~/hooks/useTaskRunsV2';
 import { getErrorState } from '~/shared/utils/error-utils';
 import { COMMIT_DETAILS_PATH } from '../../../../routes/paths';
 import { Timestamp } from '../../../../shared/components/timestamp/Timestamp';
 import { useNamespace } from '../../../../shared/providers/Namespace/useNamespaceInfo';
-import { ComponentKind } from '../../../../types';
+import { ComponentKind, ImageRepositoryVisibility } from '../../../../types';
 import { getCommitsFromPLRs } from '../../../../utils/commits-utils';
+import { getImageUrlForVisibility } from '../../../../utils/component-utils';
 import CommitLabel from '../../../Commits/commit-label/CommitLabel';
 import { useBuildLogViewerModal } from '../../../LogViewer/BuildLogViewer';
 import ScanDescriptionListGroup from '../../../PipelineRun/PipelineRunDetailsView/tabs/ScanDescriptionListGroup';
 
-type ComponentLatestBuildProps = {
-  component: ComponentKind;
-};
-
-const ComponentLatestBuild: React.FC<React.PropsWithChildren<ComponentLatestBuildProps>> = ({
-  component,
-}) => {
```

Comment:
Why is this changes needed? This is the wrong way to define types for props. Double check the code generated by AI

Reply:
@sahil143 , it is not AI change but i do. I considered this is much cleaner and simpler. 
I can change it back, but whether the saying "From a modern React TypeScript perspective, the inline prop typing approach is actually the more modern, recommended pattern. The React community has moved away from React.FC" is true?